src/WMOWithRequest/DefaultRetryError.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { View, TouchableOpacity, Text } from 'react-native';
+
+function DefaultRetryError({ text, retryText, onRetry }) {
+  return (
+    <View style={{ flex: 1, justifyContent: 'space-around', alignItems: 'center' }}>
```

Comment:
I not sure if this default components will be ever used.

What do you think about removing them and making them required options?